README.md

```diff
@@ -105,6 +105,49 @@ func testFetchConfig() async throws {
 }
 ```
 
+## Advanced Usage
+
+### Restricting the Availability of Spies
+
+If you wish, you can limit where Spyable's generated code can be used from. This can be useful if you want to prevent spies from being used in user-facing production code.
+
+To enforce this restriction, supply the `flag: String?` parameter of Spyable, like this:
+
+```swift
+@Spyable(flag: "DEBUG")
```

Comment:
I'm not 100% sure if we want to go with "flag" for this parameter name. While I like its brevity, it may not be immediately-obvious what it does when compared with other options, such as "preprocessorFlag", "compilationFlag", "compileFlag", or "buildFlag".
Any of those stick out as winners?

Reply:
What do you think about `@Spyable(behindFlag: "DEBUG")` or `behindPreprocessorFlag`?

Reply:
I went with `behindPreprocessorFlag` just to ensure it's as clear as can be in 353844a33f66426ab2f70a7fddc3d6c1b900eb98

Reply:
Great choice! Thank you! 🙌